operation/fhir-operation-cqf/README.md

```diff
@@ -0,0 +1,9 @@
+# fhir-operation-cqf
+
+This is an optional module that provides a basic implementation of some key features in the Clinical Quality Framework ([2019](http://hl7.org/fhir/us/cqfmeasures/2019May/) / [2021](http://hl7.org/fhir/us/cqfmeasures/2021May/)). This includes gathering of measure data requirements, measure evaluation, care gaps evaluation, and data submission. Operation definitions are pulled from the core FHIR R4 clinical reasoning module. There are still open items related to apply operations for ActivityDefinition and PlanDefinition apply operations. 
+
+## TODO
+* [ ] Implement Measure/$collect-data operation
+* [ ] Add support for CQL in $apply for ActivityDefinition and PlanDefinition 
+* [ ] Add support for $data-requirements the ActivityDefinition and PlanDefinition
+* [ ] Upgrade to operation definitions in May2021 cqfmeasures ballot
```

Comment:
maybe put this in a singular issue that is out there in IBM/FHIR/issues that captures or references these

Reply:
https://github.com/IBM/FHIR/issues/2613, , https://github.com/IBM/FHIR/issues/2616, https://github.com/IBM/FHIR/issues/2614